lib/ansible/modules/database/postgresql/postgresql_query.py

```diff
@@ -161,12 +163,50 @@
     postgres_common_argument_spec,
 )
 from ansible.module_utils._text import to_native
+from ansible.module_utils.six import iteritems
 
 
 # ===========================================
 # Module execution.
 #
 
+def list_to_pg_array(elem):
+    """Convert the passed list to PostgreSQL array
+    represented as a string.
+
+    Args:
+        elem (list): List that needs to be converted.
+
+    Returns:
+        elem (str): String representation of PostgreSQL array.
+    """
+    elem = str(elem).strip('[]')
```

Comment:
Why not:
```suggestion
    elem = str(elem)[1:-1]
```

Reply:
I think it is more obvious and reliable.

Reply:
For me personally, the other one is more obvious ;) But tastes definitely differ!